src/App.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import './index.css';
+import './App.css';
+import words from './words';
+import images from './images';
+import SoundEffect from './SoundEffect';
+import zvuk1 from './zvuk1.mp3';
+import zvuk2 from './zvuk2.mp3';
+
+const App = () => {
```

Comment:
App component should not contain implementation details. Create 'components' folder inside 'src' folder and separate logic into smaller pieces

Reply:
Fixed

Reply:
We seem to have distributed everything into folders, created components and broken down all the logic, or did I miss something?